sdks/python/apache_beam/yaml/yaml_mapping.py

```diff
@@ -129,14 +149,15 @@ def _as_callable(original_fields, expr, transform_name, language):
 
   _check_mapping_arguments(transform_name, **expr)
 
-  if language == "javascript":
-    return _expand_javascript_mapping_func(original_fields, **expr)
-  elif language == "python":
+  # TODO(yaml) - Disable until JsObjectWrapper can be parsed into Beam Row
+  # if language == "javascript":
```

Comment:
Maybe we could keep this under "javascript-experimental" to keep our tests running? (I would keep the error message below.)

Reply:
Done.